{
  "Faults": [
    {
      "py/object": "Fault",
      "loggedAt": "2022-09-08T17:15:00.000+0000",
      "latitude": 51.08378517274414,
      "estimatedRestoration": "2022-09-09T01:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JZ8785'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JZ8785",
      "reference": "JZ8785",
      "maxLongitude": -1.3361,
      "engineerOnSiteTime": "2022-09-08T18:13:00.000+0000",
      "minLatitude": 51.0835,
      "name": "SO22 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SO22 6NJ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3385,
              51.0847
            ],
            [
              -1.3361,
              51.0847
            ],
            [
              -1.3361,
              51.0835
            ],
            [
              -1.3385,
              51.0835
            ],
            [
              -1.3385,
              51.0847
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-09-08T18:30:00.000+0000",
      "UUID": "acde23b0-931b-4a37-8887-51c90631a833",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.3385,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.0847,
      "longitude": -1.3371173683889046,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-09-08T14:45:00.000+0000",
      "latitude": 50.71967665171275,
      "estimatedRestoration": "2022-09-09T00:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JZ8675'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JZ8675",
      "reference": "JZ8675",
      "maxLongitude": -1.1069,
      "engineerOnSiteTime": "2022-09-08T15:45:00.000+0000",
      "minLatitude": 50.7148,
      "name": "PO34 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "PO34 5BD",
        "PO34 5BL",
        "PO34 5EJ",
        "PO34 5EL",
        "PO34 5ER",
        "PO34 5ES",
        "PO34 5ET",
        "PO34 5EU",
        "PO34 5WS"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.1112,
              50.7208
            ],
            [
              -1.1069,
              50.7208
            ],
            [
              -1.1069,
              50.7148
            ],
            [
              -1.1112,
              50.7148
            ],
            [
              -1.1112,
              50.7208
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "41caa8f0-9b88-47d8-bf5c-91165680bffb",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.1112,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.7208,
      "longitude": -1.1099739524602497,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-09-09T01:11:00.000+0000",
      "latitude": 51.47665205985873,
      "estimatedRestoration": "2022-09-09T13:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JZ8946'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JZ8946",
      "reference": "JZ8946",
      "maxLongitude": -0.6166,
      "engineerOnSiteTime": "2022-09-09T02:10:00.000+0000",
      "minLatitude": 51.4735,
      "name": "SL4 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SL4 3FQ",
        "SL4 3HD",
        "SL4 3NJ",
        "SL4 3NL",
        "SL4 3NN",
        "SL4 3NT",
        "SL4 3NU",
        "SL4 3NX",
        "SL4 3NY",
        "SL4 3NZ",
        "SL4 3PA",
        "SL4 3PB",
        "SL4 3PD",
        "SL4 3PE",
        "SL4 3PF",
        "SL4 3PG",
        "SL4 3PS",
        "SL4 3PU",
        "SL4 3PZ",
        "SL4 3RZ",
        "SL4 3SA",
        "SL4 5DH"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.628,
              51.4802
            ],
            [
              -0.6166,
              51.4802
            ],
            [
              -0.6166,
              51.4735
            ],
            [
              -0.628,
              51.4735
            ],
            [
              -0.628,
              51.4802
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-09-09T02:30:00.000+0000",
      "UUID": "a90148bb-8976-42be-afea-02b55d550696",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.628,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.4802,
      "longitude": -0.6212799774084891,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-09-09T02:21:00.000+0000",
      "latitude": 50.72680665888363,
      "estimatedRestoration": "2022-09-09T10:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JZ8960'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JZ8960",
      "reference": "JZ8960",
      "maxLongitude": -1.5944,
      "minLatitude": 50.7244,
      "name": "SO41 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SO41 0NA",
        "SO41 0PP",
        "SO41 0PQ",
        "SO41 0SA",
        "SO41 0SB",
        "SO41 0UG"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.6015,
              50.7297
            ],
            [
              -1.5944,
              50.7297
            ],
            [
              -1.5944,
              50.7244
            ],
            [
              -1.6015,
              50.7244
            ],
            [
              -1.6015,
              50.7297
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "f512a087-556a-445d-87a0-e69aa3ac7a6e",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -1.6015,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 50.7297,
      "longitude": -1.5985613412831405,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-09-09T05:17:00.000+0000",
      "latitude": 51.537202868286755,
      "estimatedRestoration": "2022-09-09T08:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JZ9017'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JZ9017",
      "reference": "JZ9017",
      "maxLongitude": -0.3702,
      "minLatitude": 51.534,
      "name": "UB5 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "UB5 6AP",
        "UB5 6AR",
        "UB5 6UH",
        "UB5 6UQ",
        "UB5 6UR",
        "UB5 6UW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.3778,
              51.5411
            ],
            [
              -0.3702,
              51.5411
            ],
            [
              -0.3702,
              51.534
            ],
            [
              -0.3778,
              51.534
            ],
            [
              -0.3778,
              51.5411
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-09-09T06:15:00.000+0000",
      "UUID": "b7a24210-f46c-4cd0-8670-ce94a1f6a0b7",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.3778,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.5411,
      "longitude": -0.37269867026260284,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-09-09T04:32:00.000+0000",
      "latitude": 51.456335804673294,
      "estimatedRestoration": "2022-09-09T09:05:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JZ9005'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JZ9005",
      "reference": "JZ9005",
      "maxLongitude": -0.9695,
      "minLatitude": 51.4554,
      "name": "RG1 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG1 1EL",
        "RG1 1EP",
        "RG1 1HE",
        "RG1 1TT"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.9769,
              51.457
            ],
            [
              -0.9695,
              51.457
            ],
            [
              -0.9695,
              51.4554
            ],
            [
              -0.9769,
              51.4554
            ],
            [
              -0.9769,
              51.457
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-09-09T07:30:00.000+0000",
      "UUID": "1e60cd93-124a-4d1f-ab4f-55403269a9b2",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.9769,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.457,
      "longitude": -0.9744083045346121,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-09-09T06:22:00.000+0000",
      "latitude": 51.445782625606796,
      "estimatedRestoration": "2022-09-09T09:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JZ9050'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JZ9050",
      "reference": "JZ9050",
      "maxLongitude": -0.8638,
      "minLatitude": 51.4379,
      "name": "RG10 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG10 0SQ",
        "RG10 0SU",
        "RG10 0SX",
        "RG10 0TH"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.8944,
              51.4499
            ],
            [
              -0.8638,
              51.4499
            ],
            [
              -0.8638,
              51.4379
            ],
            [
              -0.8944,
              51.4379
            ],
            [
              -0.8944,
              51.4499
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "beb01d28-354f-4686-ab63-1c6496461e25",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -0.8944,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 51.4499,
      "longitude": -0.8682331807209432,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-09-09T04:48:00.000+0000",
      "latitude": 56.29967850711347,
      "estimatedRestoration": "2022-09-09T08:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JZ9049'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JZ9049",
      "reference": "JZ9049",
      "maxLongitude": -3.5339,
      "minLatitude": 56.2837,
      "name": "PH2 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "PH2 0RE"
      ],
      "location": {
        "coordinates": [
          [
            [
              -3.5788,
              56.3095
            ],
            [
              -3.5339,
              56.3095
            ],
            [
              -3.5339,
              56.2837
            ],
            [
              -3.5788,
              56.2837
            ],
            [
              -3.5788,
              56.3095
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-09-09T06:00:00.000+0000",
      "UUID": "af915938-26a3-4924-bf6b-639cbc84d41e",
      "region": "com.opengrid.geo.world.europe.gb",
      "minLongitude": -3.5788,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "maxLatitude": 56.3095,
      "longitude": -3.5603840536048583,
      "resolved": false
    }
  ]
}
